community/calendar/_index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "Knative Community Calendar"
+linkTitle: "Community Calendar"
+weight: 29
+showlandingtoc: "false"
+type: "docs"
+---
+
+Attend events on our Community Calendar.
```

Comment:
Do you want to add details about what's in the calendar; i.e. opportunities to learn about Knative and meet other Knative users? You might also mention that events need not be organized by the Knative organization to be added to the calendar.

Reply:
Thank you I was looking for type of info to put here @evankanderson 

Should we add something if you want to make us aware of an event to add to the Calendar email us (should I use steering email?), posted in #communtiy Slack channel, or mention KnativeProject@ in Twitter ?